### ISSUE TITLE
No use of divs

### ISSUE THREAD

**Issue by farishkash** [BARKINGnoMAD/prj-rev-bwfs-dasmoto#1]
So in the first project we review, I look for divs.

The idea behind a div is that they are designed to divide your website into blocks of content to be used to better organize and manage. As you progress you will see the use of divs quite frequently.

Example below I rewrote this section wrapping it in a div.

```
<div class="someclassname">

<h2 id="brushes">Brushes</h2>
  <img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-2/hacksaw.jpeg">
  <h3>Hacksaw Brushes</h3>
  <p>Made of the highest quality oak, Hacksaw brushes are known for their weight and ability to hold paint in large amounts. Available in different sizes. <span class="price">Starting at $3.00 / brush.</span></p>

</div>

```

**Comment by BARKINGnoMAD** [comment]
Are you saying I should declare divs even though I don't have any attributes to apply to them? Just for organizational purposes? Meaning, should I put them in my html even though I don't use them in my css file?

**Comment by farishkash** [comment]
Yes and this is more to prepare you for what is coming up next, I am just pushing you into these habits.  As you will see soon, you will be using divs a lot, basically non stop.

The problem with this project that even though it is for introductory purposes, it is design like it was made in the 90s and I feel needs to go just a little bit deeper.

I get the point of why you all do it, this is more of my own opinion on it.
